web-client/src/app/state/session-xrpl.service.ts

```diff
@@ -161,9 +162,10 @@ export class SessionXrplService {
     );
 
     const txResponse = await this.submitTransaction(txnSignedEncoded);
-    const txSucceeded = checkTxResponseSucceeded(txResponse);
+    await this.loadAccountData();
```

Comment:
Do we need to call `loadAccounData()`?

Reply:
This might be causing the Auto Logout problems.

Reply:
Nvm, this is not the cause of the auto logout feature. But do we need to call `loadAcountData()`?